extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -35,6 +35,7 @@ import {LoadingSpinner} from './loading-spinner';
 import {MediaPool} from './media-pool';
 import {PageScalingService} from './page-scaling';
 import {Services} from '../../../src/services';
+import {VideoServiceSync} from '../../../src/service/video-service-sync-impl';
```

Comment:
This is okay if it somehow makes your life easier, but generally speaking only P0 issues need to be resolved in 0.1.  Any new features and fixes can happen in 1.0.

Reply:
I can revert 0.1 but that means supporting two delegation APIs at once.

If stability of `0.1` is a concern I don't mind doing the change (noting that 0.1 will still be bloated by the video service).